File: 10 - Python/Exercícios/03 - Estruturas Repetitivas/For/Exercício 07.py
#Exercício - Divisão

n = int(input("Quantos casos você vai digitar? "))

for i in range(n):

    x = float(input("Entre com o numerador: "))
    y = float(input("Entre com o denominador: "))

    if y != 0:
        divisao = x / y
        print("Divisão =", divisao)
        print("")
    else:
        print("Divisão impossível!")
        print("")

File: 06 - C/Exercícios/03 - Estruturas Repetitivas/For/Exercício 07.c
//Exercício - Divisão

#include <stdio.h>

int main(){

    int i, N;
    double x, y, divisao;

    printf("Quantos casos voce vai digitar? ");
    scanf("%d", &N);

    for(i = 1; i <= N; i++){

        printf("\nEntre com o numerador: ");
        scanf("%lf", &x);

        printf("Entre com o denominador: ");
        scanf("%lf", &y);

        if(y != 0){
            divisao = (double) x / y;
            printf("Divisao = %.2lf\n", divisao);
        }
        else{
            printf("Divisao impossivel\n");
        }
    }

    return 0;
    
}